front-end/src/components/cookie/CookieView.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Modal, Button } from "react-bootstrap";
+
+import "./CookieView.css";
+
+class CookieView extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            show: true
+        };
+    
+        this.handleClose = this.handleClose.bind(this);
+        this.isCookieSet = cookiePolicy => cookiePolicy && cookiePolicy === "true" ? true : false;
+        this.cookies = this.props.cookies;
+    }
+
+    handleClose(isPolicyAccepted) {
+        this.setState({ show: false });
+        if (isPolicyAccepted && isPolicyAccepted === true) {
+            if (this.cookies) {
+                this.cookies.set("policy", isPolicyAccepted, { maxAge: (10 * 365 * 24 * 60 * 60 )});
+            }
+        }
+    }
+
+    render() {
+        const cookiePolicy = this.cookies.get('policy');
+        const showCookieDiv = !this.isCookieSet(cookiePolicy) && this.state.show;
+
+        return (
+            <Modal show={showCookieDiv} onHide={this.handleClose} bsClass="cookie-modal">
+                <Modal.Header closeButton>
+                    <Modal.Title>Cookie Policy</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    The Vantaa Feed website uses cookies to ensure you receive the best experience on our awesome site! 
```

Comment:
i18n?

Reply:
Yes, I think internationalization should be done across the whole project as a separate ticket. I will create one for that.